### PR TITLE
Move docs

### PR DESCRIPTION
**Pull request by omsenjalia**
This is to make it easier for users to read

### CODE REVIEW

**Comment by TerraPlayz** [comment]
@MaskDuck and @WilliamDavidHarrison my PR is ready. Please let me know on discord and suggest any changes that you can improve. (Please keep credits section)

**Review by wdhdev — CHANGES_REQUESTED** [review]
~~Please make all of the files and folders lowercase.~~ Done.

File: Docs/credits/Index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax), [Discord](https://discordapp.com/users/730358289849778186)
+
+### Maintiners
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:5]: Please alphabetise this section and remove the Discord accounts for privacy reasons.

File: Docs/credits/Index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- **TheBotlyNoob:** [GitHub](https://github.com/TheBotlyNoob),[Discord](https://discordapp.com/users/488802888928329753)
+
+- **Hackermon:** [GitHub](https://github.com/hackermondev),[Discord](https://discordapp.com/users/1044657759066525777)
+
+## Helpers
+
+- **Vaibhav:** [GitHub](https://github.com/VaibhavSys),[Discord](https://discordapp.com/users/914452175839723550)
+
+### Mods
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:22]:
```suggestion
```

File: Docs/credits/Index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- **Omxpro** [GitHub](https://github.com/TerraPlayz),[Discord](https://discordapp.com/users/248470317540966443)
+
+
+##### These are all the people that work with is-a.dev. They are all the people that made is-a.dev in it's glory as it is now. so please take a moment to thank them and appreciate their hard efforts. Thanks for using is-a.dev.
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:26]: Seems a bit "tacky"

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: Docs/credits/Index.md (hunk 3 above)
Comment (TerraPlayz) [reply Docs/credits/Index.md:26]: Tacky?

File: Docs/credits/Index.md (hunk 2 above)
Comment (TerraPlayz) [reply Docs/credits/Index.md:22]: I am a mod that's why there is a mod section

File: Docs/credits/Index.md (hunk 1 above)
Comment (TerraPlayz) [reply Docs/credits/Index.md:5]: Alright

File: Docs/credits/Index.md (hunk 2 above)
Comment (WilliamDavidHarrison) [reply Docs/credits/Index.md:22]: @TerraPlayz That's for the Discord server

File: Docs/credits/Index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax)
+
+### Maintiners
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:5]:
```suggestion
### Maintainers
```

File: Docs/credits/Index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:3]:
```suggestion
- **Phenax**: [GitHub](https://github.com/Phenax)
```

File: Docs/credits/Index.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+### Maintiners
+- **MaskDuck:** [GitHub](https://github.com/MaskDuck)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:6]:
```suggestion
- **MaskDuck**: [GitHub](https://github.com/MaskDuck)
```

File: Docs/credits/Index.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+- **William:** [GitHub](https://github.com/WilliamDavidHarrison)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:8]:
```suggestion
- **William**: [GitHub](https://github.com/WilliamDavidHarrison)
```

File: Docs/credits/Index.md [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Andrew:** [GitHub](https://github.com/andrewstech)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:10]:
```suggestion
- **Andrew**: [GitHub](https://github.com/andrewstech)
```

File: Docs/credits/Index.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Mahir/Molai** [GitHub](https://github.com/mtgsquad)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:12]:
```suggestion
- **Mahir**: [GitHub](https://github.com/mtgsquad)
```

File: Docs/credits/Index.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+- **TheBotlyNoob:** [GitHub](https://github.com/TheBotlyNoob)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:14]:
```suggestion
- **TheBotlyNoob**: [GitHub](https://github.com/TheBotlyNoob)
```

File: Docs/credits/Index.md [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Hackermon:** [GitHub](https://github.com/hackermondev)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:16]:
```suggestion
- **Hackermon**: [GitHub](https://github.com/hackermondev)
```

File: Docs/credits/Index.md [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+
+## Helpers
+
+- **Vaibhav:** [GitHub](https://github.com/VaibhavSys)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:20]:
```suggestion
- **Vaibhav**: [GitHub](https://github.com/VaibhavSys)
```

File: Docs/credits/Index.md [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+
+### Mods
+- **Omxpro** [GitHub](https://github.com/TerraPlayz)
```
Comment (WilliamDavidHarrison) [inline Docs/credits/Index.md:23]:
```suggestion
- **Om**: [GitHub](https://github.com/TerraPlayz)
```

File: Docs/credits/Index.md (hunk 3 above)
Comment (WilliamDavidHarrison) [reply Docs/credits/Index.md:26]: @TerraPlayz Out of place, sort of.

File: Docs/github/Index.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
+# Setting up GitHub pages with is-a.dev subdomain
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:1]:
```suggestion
# GitHub Pages
```

File: Docs/github/Index.md [hunk 15]
```diff
@@ -0,0 +1,25 @@
...
+
+## Creating a GitHub pages repository
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:3]:
```suggestion
## 1. Create a repository
```

File: Docs/github/Index.md [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+You can create a GitHub pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub pages, please read through [their guide](https://guides.github.com/features/pages).
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:4]:
```suggestion
**If you already have a repository with GitHub Pages, you can skip this step.**

You can create a GitHub Pages website by creating a repository with the name `<your-github-username>.github.io`. For more information about GitHub Pages, please read through the [GitHub Pages documentation](https://guides.github.com/features/pages).
```

File: Docs/github/Index.md [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+
+## Creating the domain file
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:6]:
```suggestion
## 2. Registering an is-a.dev subdomain
```

File: Docs/github/Index.md [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:7]:
```suggestion
Create a JSON file inside `domains` directory (`domains/<subdomain-name>.json`) with the following content:
```

File: Docs/github/Index.md [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<github-username>/<github-username>.github.io",
+    "owner": {
+        "username": "<github-username>",
+        "email": "email@address",
+        "twitter": "<twitter-username>"
+    },
+    "record": {
+        "CNAME": "<username>.github.io"
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:18]:
```suggestion
        "CNAME": "<your-github-username>.github.io"
```

File: Docs/github/Index.md [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+## Creating the domain file
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
+```json 
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<github-username>/<github-username>.github.io",
+    "owner": {
+        "username": "<github-username>",
+        "email": "email@address",
+        "twitter": "<twitter-username>"
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:15]:
```suggestion
```

Not required.

File: Docs/github/Index.md [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+
+## Creating the domain file
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
+```json 
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<github-username>/<github-username>.github.io",
+    "owner": {
+        "username": "<github-username>",
+        "email": "email@address",
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:14]:
```suggestion
        "email": "email@address.tld"
```

File: Docs/github/Index.md [hunk 22]
```diff
@@ -0,0 +1,25 @@
...
+You can create a GitHub pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub pages, please read through [their guide](https://guides.github.com/features/pages).
+
+## Creating the domain file
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
+```json 
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<github-username>/<github-username>.github.io",
+    "owner": {
+        "username": "<github-username>",
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:13]:
```suggestion
        "username": "<your-github-username>",
```

File: Docs/github/Index.md [hunk 23]
```diff
@@ -0,0 +1,25 @@
...
+
+## Creating a GitHub pages repository
+You can create a GitHub pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub pages, please read through [their guide](https://guides.github.com/features/pages).
+
+## Creating the domain file
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
+```json 
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/<github-username>/<github-username>.github.io",
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:11]:
```suggestion
    "repo": "https://github.com/<your-github-username>/<your-github-username>.github.io",
```

File: Docs/github/Index.md [hunk 24]
```diff
@@ -0,0 +1,25 @@
+# Setting up GitHub pages with is-a.dev subdomain
+
+## Creating a GitHub pages repository
+You can create a GitHub pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub pages, please read through [their guide](https://guides.github.com/features/pages).
+
+## Creating the domain file
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
+```json 
+{
+    "description": "Describe the use of this subdomain",
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:10]:
```suggestion
    "description": "A description of your website",
```

File: Docs/github/Index.md [hunk 25]
```diff
@@ -0,0 +1,25 @@
...
+        "email": "email@address",
+        "twitter": "<twitter-username>"
+    },
+    "record": {
+        "CNAME": "<username>.github.io"
+    }
+} 
+```
+
+## Configuring
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:23]:
```suggestion
## 3. Configuring your subdomain
```

File: Docs/github/Index.md [hunk 26]
```diff
@@ -0,0 +1,25 @@
...
+- After the pull request is merged, you will see a **404** error on `<your-subdomain>.is-a.dev`. To fix this, go to your GitHub pages repository's **Settings > GitHub pages > Custom domain** and add `<your-subdomain>.is-a.dev` in the given field. _Only do this **after** your pull request is merged._
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:24]:
```suggestion
- After your pull request is merged, you will see a **404** error on your subdomain: `<subdomain>.is-a.dev`. To fix this, go to your repository's **Settings > Pages > Custom domain** and add `<subdomain>.is-a.dev` in the given field. __Only do this **after** your pull request is merged.__
```

File: Docs/github/Index.md [hunk 27]
```diff
@@ -0,0 +1,25 @@
...
+- Check the **Enforce HTTPS** checkbox below the custom domain input.
```
Comment (WilliamDavidHarrison) [inline Docs/github/Index.md:25]:
```suggestion
- Check the `Enforce HTTPS` checkbox below the custom domain input, after the SSL certificate has been generated.
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Comment by TerraPlayz** [comment]
I can't commit anything you suggest as it says outdated

File: docs/credits/index.md [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+
+- **TheBotlyNoob:** [GitHub](https://github.com/TheBotlyNoob)
+
+- **Hackermon:** [GitHub](https://github.com/hackermondev)
+
+## Helpers
+
+- **Vaibhav:** [GitHub](https://github.com/VaibhavSys)
+
+### Mods
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:22]:
```suggestion
```

**Comment by WilliamDavidHarrison** [comment]
> I can't commit anything you suggest as it says outdated

Fixing now.

File: docs/credits/index.md [hunk 29]
```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:3]:
```suggestion
- **Phenax**: [GitHub](https://github.com/phenax)
```

File: docs/credits/index.md [hunk 30]
```diff
@@ -0,0 +1,26 @@
...
+
+### Maintainers
+- **MaskDuck:** [GitHub](https://github.com/MaskDuck)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:6]:
```suggestion

- **MaskDuck**: [GitHub](https://github.com/MaskDuck)
```

File: docs/credits/index.md [hunk 31]
```diff
@@ -0,0 +1,26 @@
...
+
+- **William:** [GitHub](https://github.com/WilliamDavidHarrison)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:8]:
```suggestion
- **William**: [GitHub](https://github.com/WilliamDavidHarrison)
```

File: docs/credits/index.md [hunk 32]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Andrew:** [GitHub](https://github.com/andrewstech)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:10]:
```suggestion
- **Andrew**: [GitHub](https://github.com/andrewstech)
```

File: docs/credits/index.md [hunk 33]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Mahir/Molai** [GitHub](https://github.com/mtgsquad)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:12]:
```suggestion
- **Mahir**: [GitHub](https://github.com/mtgsquad)
```

File: docs/credits/index.md [hunk 34]
```diff
@@ -0,0 +1,26 @@
...
+
+- **TheBotlyNoob:** [GitHub](https://github.com/TheBotlyNoob)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:14]:
```suggestion
- **TheBotlyNoob**: [GitHub](https://github.com/TheBotlyNoob)
```

File: docs/credits/index.md [hunk 35]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Hackermon:** [GitHub](https://github.com/hackermondev)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:16]:
```suggestion
- **Hackermon**: [GitHub](https://github.com/hackermondev)
```

File: docs/credits/index.md [hunk 36]
```diff
@@ -0,0 +1,26 @@
...
+
+## Helpers
+
+- **Vaibhav:** [GitHub](https://github.com/VaibhavSys)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:20]:
```suggestion
- **Vaibhav**: [GitHub](https://github.com/VaibhavSys)
```

File: docs/credits/index.md [hunk 37]
```diff
@@ -0,0 +1,26 @@
...
+
+### Mods
+- **Omxpro** [GitHub](https://github.com/TerraPlayz)
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:23]:
```suggestion
- **Om**: [GitHub](https://github.com/TerraPlayz)
```

File: docs/credits/index.md [hunk 38]
```diff
@@ -0,0 +1,26 @@
...
+
+
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:25]:
```suggestion
```

File: docs/credits/index.md [hunk 39]
```diff
@@ -0,0 +1,26 @@
...
+##### These are all the people that work with is-a.dev. They are all the people that made is-a.dev in it's glory as it is now. so please take a moment to thank them and appreciate their hard efforts. Thanks for using is-a.dev.
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:26]:
```suggestion
```

Seems out of place, almost like a funeral speech

**Review by wdhdev — CHANGES_REQUESTED** [review]
Credits page

File: docs/credits/index.md [hunk 40]
```diff
@@ -0,0 +1,26 @@
+### Owner
+
+- **Phenax:** [GitHub](https://github.com/Phenax)
+
+### Maintainers
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:5]:
```suggestion
# Maintainers
```

File: docs/credits/index.md [hunk 41]
```diff
@@ -0,0 +1,26 @@
+### Owner
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:1]:
```suggestion
# Owner
```

File: docs/credits/index.md [hunk 42]
```diff
@@ -0,0 +1,26 @@
...
+
+- **Andrew:** [GitHub](https://github.com/andrewstech)
+
+- **Mahir/Molai** [GitHub](https://github.com/mtgsquad)
+
+- **TheBotlyNoob:** [GitHub](https://github.com/TheBotlyNoob)
+
+- **Hackermon:** [GitHub](https://github.com/hackermondev)
+
+## Helpers
```
Comment (WilliamDavidHarrison) [inline docs/credits/index.md:18]:
```suggestion
# Helpers
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Few more for the credits page

File: docs/github/index.md [hunk 43]
```diff
@@ -0,0 +1,25 @@
+# Setting up GitHub pages with is-a.dev subdomain
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:1]:
```suggestion
# GitHub Pages
```

File: docs/github/index.md [hunk 44]
```diff
@@ -0,0 +1,25 @@
...
+
+## Creating a GitHub pages repository
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:3]:
```suggestion
## 1. Creating a repository
```

File: docs/github/index.md [hunk 45]
```diff
@@ -0,0 +1,25 @@
...
+You can create a GitHub pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub pages, please read through [their guide](https://guides.github.com/features/pages).
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:4]:
```suggestion
**If you have already created a repository and setup GitHub Pages, you can skip this step.

You can create a GitHub Pages website by creating a repository with the name `<github-username>.github.io`. For more information about GitHub Pages, please read through the GitHub Pages [docs](https://guides.github.com/features/pages).
```

File: docs/github/index.md [hunk 46]
```diff
@@ -0,0 +1,25 @@
...
+
+## Creating the domain file
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:6]:
```suggestion
## 2. Register a subdomain
```

File: docs/github/index.md [hunk 47]
```diff
@@ -0,0 +1,25 @@
...
+Create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:7]:
```suggestion
First, create a JSON file inside `domains` directory (`domains/<subdomain>.json`) with the following content:

```

File: docs/github/index.md [hunk 48]
```diff
@@ -0,0 +1,25 @@
...
+```json 
+{
+    "description": "Describe the use of this subdomain",
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:10]:
```suggestion
    "description": "A description of your website",
```

File: docs/github/index.md [hunk 49]
```diff
@@ -0,0 +1,25 @@
...
+    "repo": "https://github.com/<github-username>/<github-username>.github.io",
+    "owner": {
+        "username": "<github-username>",
+        "email": "email@address",
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:14]:
```suggestion
        "email": "email@address.tld"
```

File: docs/github/index.md [hunk 50]
```diff
@@ -0,0 +1,25 @@
...
+        "twitter": "<twitter-username>"
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:15]:
```suggestion
```

Remove as it is not a requirement, only email is.

File: docs/github/index.md [hunk 51]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    "record": {
+        "CNAME": "<username>.github.io"
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:18]:
```suggestion
        "CNAME": "<github-username>.github.io"
```

File: docs/github/index.md [hunk 52]
```diff
@@ -0,0 +1,25 @@
...
+    }
+} 
+```
+
+## Configuring
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:23]:
```suggestion
## 3. Configure your subdomain
```

File: docs/github/index.md [hunk 53]
```diff
@@ -0,0 +1,25 @@
...
+- After the pull request is merged, you will see a **404** error on `<your-subdomain>.is-a.dev`. To fix this, go to your GitHub pages repository's **Settings > GitHub pages > Custom domain** and add `<your-subdomain>.is-a.dev` in the given field. _Only do this **after** your pull request is merged._
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:24]:
```suggestion
- **After your pull request is merged**, you will see a **404** error on `<subdomain>.is-a.dev`. To fix this, go to your repository and go to: **Settings > GPages > Custom Domain** and add `<subdomain>.is-a.dev` in the given field. __Only do this **after** your pull request is merged.__
```

File: docs/github/index.md [hunk 54]
```diff
@@ -0,0 +1,25 @@
...
+- Check the **Enforce HTTPS** checkbox below the custom domain input.
```
Comment (WilliamDavidHarrison) [inline docs/github/index.md:25]:
```suggestion
- Check the **Enforce HTTPS** checkbox below the custom domain input, after the SSL certificate has been generated.
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
GitHub Pages page

**Review by MaskDuck — DISMISSED** [review]
please use submodules

**Comment by TerraPlayz** [comment]
> please use submodules
> 

What?

**Comment by WilliamDavidHarrison** [comment]
> > > please use submodules
> > 
> > 
> > What?
> 
> https://git-scm.com/book/en/v2/Git-Tools-Submodules

You can publish to GitHub Pages from the `docs` folder, so this isn't required.

**Review by wdhdev — APPROVED** [review]

**Review by MaskDuck — CHANGES_REQUESTED** [review]
point still stands

**Review by wdhdev — CHANGES_REQUESTED** [review]
I don't think we will be moving the docs at this stage, so I am closing this PR.

**Pull request closed without merging**